webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -61,9 +61,10 @@ export default class CreateIssueModal extends PureComponent {
             e.preventDefault();
         }
 
-        if (!this.validator.validate() || !this.state.issueTitle) {
+        const isValidTitle = this.state.issueTitle.trim().length !== 0;
+        if (!this.validator.validate() || !isValidTitle) {
             this.setState({
-                issueTitleValid: Boolean(this.state.issueTitle),
+                issueTitleValid: Boolean(isValidTitle),
```

Comment:
@cyrusjc Thanks for the PR.
Here isValidTitle is already going to be a boolean value. Is there any need to type caste into a Boolean again?

Reply:
Ah no particular reason other than a small oversight on my part.